test/run-tests.sh

```diff
@@ -128,4 +128,8 @@ fi
 echo "Skipping following test :: ${TEST_TO_SKIP}"
 
 # Running only smoke test cases by default or value passed through TEST_FOCUS env variable. To run different test packages add/remove path from focus argument or TEST_FOCUS variable
-ginkgo -v -progress -r -keepGoing -nodes=${CLUSTER_NODES} --noisyPendings=false --reportPassed --focus="${TEST_TO_RUN}" --skip="${TEST_TO_SKIP}" ${topdir}/test/ -- -commit-hash=${COMMIT_HASH} -operator-image=${PRIVATE_SPLUNK_OPERATOR_IMAGE}  -splunk-image=${PRIVATE_SPLUNK_ENTERPRISE_IMAGE}
\ No newline at end of file
+ginkgo -v -progress -r -keepGoing -nodes=${CLUSTER_NODES} --noisyPendings=false --reportPassed --focus="${TEST_TO_RUN}" --skip="${TEST_TO_SKIP}" ${topdir}/test/ -- -commit-hash=${COMMIT_HASH} -operator-image=${PRIVATE_SPLUNK_OPERATOR_IMAGE}  -splunk-image=${PRIVATE_SPLUNK_ENTERPRISE_IMAGE}
+
+if [[ "${DEBUG}" != "True" ]]; then
+  tools/cleanup.sh
```

Comment:
We call this script as part of `Teardown()` too, under what conditions will we need to run here?

Reply:
The Teardown() part was a patch for app-framework branch only to cleanup before running next test. On develop we are running every test on its own namespace. I was expecting to undo the Teardown() change when we bring develop changes back to appframework. 

This would we run after the ginkgo run on like 131 is complete.

Reply:
Thanks. Can we create a task to followup the revert of Teardown() change?

Reply:
should we add pv deletion to cleanup ?